RELEASE_NOTES.md

```diff
@@ -1,11 +1,20 @@
-## Regen Ledger v2.1.0
+## Regen Ledger v3.0.0
 
-### IBC Patch Upgrade
+Regen Ledger `v3.0.0` adds basket functionality to the ecocredit module.
 
-Regen Ledger `v2.1.0` includes an important fix for Regen Mainnet which recently upgraded to Regen Ledger `v2.0.0`.  In the `v2.0.0` upgrade, a bug was introduced that made all new IBC transactions fail to be processed.
+### Baskets
 
-This release (`v2.1.0`) hard codes an emergency height-based upgrade which introduces a consensus-breaking change at height `3126912` that resolves the issue. All production validators and full-nodes must update to `v2.1.0` prior to the upgrade height (estimated for Friday Nov 26, 17:00 UTC).
+Regen Ledger `v3.0.0` updates the ecocredit module to include basket functionality, enabling the aggregation of heterogeneous ecosystem service credits into baskets. Credits from different credit classes and batches that meet a defined criteria can be deposited within a basket in exchange for basket tokens. The basket tokens are fully fungible with other tokens from the same basket, and are tracked in the standard bank module, which means these assets will be made visible in wallets like Keplr, and also able to be transferred via IBC to other chains in the Cosmos ecosystem. Each basket token can later be redeemed for an underlying ecocredit from the given basket, and the ecocredits received may be retired by the account receiving them (for offsetting emissions).
```

Comment:
```suggestion
Regen Ledger `v3.0.0` updates the ecocredit module to include basket functionality, enabling the aggregation of heterogeneous ecosystem service credits into baskets. Credits from different credit classes and batches that meet a defined criteria can be deposited within a basket in exchange for basket tokens. The basket tokens are fully fungible with other tokens from the same basket, and are tracked in the standard bank module, which means these assets will be made visible in wallets like Keplr, and also able to be transferred via IBC to other chains in the Cosmos ecosystem. Basket tokens can later be redeemed for underlying ecocredits from the given basket, and the ecocredits received may be retired by the account receiving them (for offsetting emissions).
```
I think that makes it sound like it's a 1:1 relationship, but you could have, say, 500 tokens : 1 credit ratio

Reply:
Don't we always have a 1-1 ratio (modulo the basket exponent)?

Reply:
ah right, for non prefixed tokens. im guessing users wont see the prefixed tokens ever so ill mark this as resolved